# helpful_links

[![Check Links](https://github.com/paulstothard/helpful_links/actions/workflows/links.yml/badge.svg)](https://github.com/paulstothard/helpful_links/actions/workflows/links.yml) [![GitHub Super-Linter](https://github.com/paulstothard/helpful_links/actions/workflows/linter.yml/badge.svg)](https://github.com/paulstothard/helpful_links/actions/workflows/linter.yml) [![Generate TOC](https://github.com/paulstothard/helpful_links/actions/workflows/toc.yml/badge.svg)](https://github.com/paulstothard/helpful_links/actions/workflows/toc.yml)

Links to resources related to bioinformatics and data analysis.

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
**Table of Contents**

- [helpful\_links](#helpful_links)
  - [ATAC-seq](#atac-seq)
  - [Audiovisual](#audiovisual)
  - [BAM and SAM](#bam-and-sam)
  - [ChIP-seq](#chip-seq)
  - [Containers](#containers)
  - [Courses and training](#courses-and-training)
  - [Data sharing and management](#data-sharing-and-management)
  - [Data visualization](#data-visualization)
  - [Diagrams and flowcharts](#diagrams-and-flowcharts)
  - [Document conversion and manipulation](#document-conversion-and-manipulation)
  - [Documentation creation and publishing](#documentation-creation-and-publishing)
  - [EMBL-EBI](#embl-ebi)
  - [Genome alignment and comparison](#genome-alignment-and-comparison)
  - [Genome annotation and sequence characterization](#genome-annotation-and-sequence-characterization)
  - [Genome assembly](#genome-assembly)
  - [Genome visualization](#genome-visualization)
  - [GTF and GFF](#gtf-and-gff)
  - [GWAS](#gwas)
  - [Images](#images)
  - [Machine learning](#machine-learning)
  - [Metagenomics](#metagenomics)
  - [Methyl-seq](#methyl-seq)
  - [Multiomics](#multiomics)
  - [NCBI](#ncbi)
  - [Phylogenetics and phylogenomics](#phylogenetics-and-phylogenomics)
  - [Population genetics and conservation](#population-genetics-and-conservation)
  - [Programming](#programming)
  - [Raw sequence data processing and QC](#raw-sequence-data-processing-and-qc)
  - [RNA-Seq](#rna-seq)
  - [Sequence read alignment](#sequence-read-alignment)
  - [Sequence searching and clustering](#sequence-searching-and-clustering)
  - [Sequence utilities](#sequence-utilities)
  - [Software development tools](#software-development-tools)
  - [Statistics](#statistics)
  - [Tabular data](#tabular-data)
  - [Utilities](#utilities)
  - [Variant identification and analysis](#variant-identification-and-analysis)
  - [VCF files](#vcf-files)
  - [Vim](#vim)
  - [Workflow development and workflows](#workflow-development-and-workflows)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

## ATAC-seq

- [nf-core/atacseq](https://github.com/nf-core/atacseq) - ATAC-seq peak-calling, QC and differential analysis pipeline.

## Audiovisual

- [ffmpeg](https://ffmpeg.org) - A complete, cross-platform solution to record, convert and stream audio and video.
- [ffmprovisr](https://amiaopensource.github.io/ffmprovisr/) - A tool to help you build complex FFmpeg commands without writing a single line of code.

## BAM and SAM

- [alimanfoo/pysamstats](https://github.com/alimanfoo/pysamstats) - Reports simple statistics for genome positions based on sequence alignments from a SAM or BAM file.
- [genome/bam-readcount](https://github.com/genome/bam-readcount) - Generates low-level information about sequencing data at specific nucleotide positions in a BAM or CRAM file.
- [pysam-developers/pysam](https://github.com/pysam-developers/pysam) - A Python module for reading and manipulating SAM/BAM/VCF/BCF files.
- [samtools/samtools](https://github.com/samtools/samtools) - Tools for manipulating next-generation sequencing data.
- [shiquan/bamdst](https://github.com/shiquan/bamdst) - Generate BAM file statistics.

## ChIP-seq

- [nf-core/chipseq](https://github.com/nf-core/chipseq) - ChIP-seq peak-calling, QC and differential analysis pipeline.

## Containers

- [Docker Hub](https://hub.docker.com) - Find and share Docker container images.
- [Quay](https://quay.io) - Find and share container images.

## Courses and training

- [Canadian Bioinformatics Workshops](https://bioinformatics.ca/workshops/current-workshops/) - The Canadian Bioinformatics Workshops (CBW) offered through bioinformatics.ca focuses on training in leading technologies and the latest methods being used in computational biology to work with these data.
- [matloff/fasteR](https://github.com/matloff/fasteR) - A thorough and thoughtful introduction to R for non-programmers.
- [raivivek/awesome-biology](https://github.com/raivivek/awesome-biology) - Learning resources, research papers, tools, and other resources across different fields of biology.
- [sib-swiss/training-collection](https://github.com/sib-swiss/training-collection) - Bioinformatics training materials.

## Data sharing and management

- [datalad/datalad](https://github.com/datalad/datalad) - Keep code, data, containers under control with git and git-annex.
- [Zenodo](https://zenodo.org) - Store research-related data, software, and reports and make them citable using a DOI.

## Data visualization

- [arvestad/alv](https://github.com/arvestad/alv) - A console-based alignment viewer.
- [ChartsCSS/charts.css](https://github.com/ChartsCSS/charts.css) - Open source CSS framework for data visualization.
- [cxli233/FriendsDontLetFriends](https://github.com/cxli233/FriendsDontLetFriends) - Examples of good and bad practices in data visualization.
- [cytoscape/cytoscape.js](https://github.com/cytoscape/cytoscape.js) - Graph theory (network) library for visualisation and analysis.
- [davidgohel/ggiraph](https://github.com/davidgohel/ggiraph/) - Create dynamic ggplot graphs with tooltips, hover effects and JavaScript actions.
- [dreamRs/esquisse](https://github.com/dreamRs/esquisse) - RStudio add-in to make plots interactively with ggplot2.
- [ewels/MultiQC](https://github.com/ewels/MultiQC) - Aggregate results from bioinformatics analyses across many samples into a single report.
- [GenomeVIS USASK](https://genomevis.usask.ca) - A variety of browser-based visualization tools to support genomics research.
- [hms-dbmi/UpSetR](https://github.com/hms-dbmi/UpSetR) - An R implementation of the UpSet set visualization technique.
- [jbkunst/highcharter](https://github.com/jbkunst/highcharter/) - AN R wrapper for the Highcharts javascript library and its modules. Highcharts is very flexible and customizable javascript charting library and it has a great and powerful API.
- [krassowski/complex-upset](https://github.com/krassowski/complex-upset) - A library for creating complex UpSet plots with ggplot2 geoms.
- [mw201608/SuperExactTest](https://github.com/mw201608/SuperExactTest) - Statistical testing and visualization of intersections among multiple sets.
- [plotly/plotly.py](https://github.com/plotly/plotly.py) - An interactive, open-source, and browser-based graphing library for Python.
- [plotly/plotly.R](https://github.com/plotly/plotly.R) - An interactive, open-source, and browser-based graphing library for R.
- [R CHARTS](https://r-charts.com) - Code examples of R graphs made with base R graphics, ggplot2 and other packages.
- [rich-iannone/DiagrammeR](https://github.com/rich-iannone/DiagrammeR) - Graph and network visualization using tabular data in R.
- [seaborn](https://seaborn.pydata.org) - A Python data visualization library that provides a high-level interface for drawing attractive and informative statistical graphics.
- [simonw/datasette](https://github.com/simonw/datasette) - A tool for exploring and publishing data. It helps people take data of any shape or size and publish that as an interactive, explorable website and accompanying API.
- [slowkow/ggrepel](https://github.com/slowkow/ggrepel) - Provides geoms for ggplot2 to repel overlapping text labels.
- [smin95/smplot2](https://github.com/smin95/smplot2) - An R package for statistical data visualization that complements ggplot2.
- [taiyun/corrplot](https://github.com/taiyun/corrplot) - A visual exploratory tool on correlation matrix.
- [thackl/gggenomes](https://github.com/thackl/gggenomes) - A versatile graphics package for comparative genomics.
- [thomasp85/patchwork](https://github.com/thomasp85/patchwork) - The goal of patchwork is to make it ridiculously simple to combine separate ggplots into the same graphic.
- [wilkox/gggenes](https://github.com/wilkox/gggenes) - Draw gene arrow maps in ggplot2.

## Diagrams and flowcharts

- [jgraph/drawio](https://github.com/jgraph/drawio) - A configurable diagramming application.
- [mermaid-js/mermaid](https://github.com/mermaid-js/mermaid) - Generation of diagrams and flowcharts from text.

## Document conversion and manipulation

- [jgm/pandoc](https://github.com/jgm/pandoc) - A library for converting from one markup format to another, and a command-line tool that uses this library.
- [pansapiens/standalone_html.py](https://gist.github.com/pansapiens/110431456e8a4ba4f2eb) - Convert HTML to a self contained file with inline Base64 encoded PNG images.
- [paulstothard/document-builder](https://github.com/paulstothard/document-builder) - Produces nicely formatted PDF and HTML documents from Markdown source files.
- [py-pdf/pypdf](https://github.com/py-pdf/pypdf) - A pure-python PDF library capable of splitting, merging, cropping, and transforming the pages of PDF files.

## Documentation creation and publishing

- [Antora](https://antora.org/) - A modular documentation site generator designed for technical documentation sites with customizable AsciiDoc structure for managing large-scale projects.
- [Astro](https://astro.build/) - A modern static site generator allowing the creation of websites using a mix of static and dynamic components, supporting various rendering engines and frameworks.
- [Docusaurus](https://docusaurus.io/) - An open-source framework for building static websites, particularly for documentation, with simple configuration, Markdown support, versioning, search, and localization.
- [GitBook](https://www.gitbook.com/) - An online platform for creating and hosting documentation with version control, team collaboration, and integrations for a collaborative writing environment.
- [honkit/honkit](https://github.com/honkit/honkit) - A command-line tool and Node.js library based on GitBook for building  online documentation or ebooks using Markdown files.
- [Jekyll](https://jekyllrb.com/) - A static site generator for creating websites or blogs using plain text files and Markdown syntax with customizable templates.
- [JupyterBook](https://jupyterbook.org/) - An open-source tool for building interactive, publication-quality books and documentation using Jupyter Notebooks and Markdown, incorporating code, visualizations, and interactive elements.
- [Kozea/WeasyPrint](https://github.com/Kozea/WeasyPrint) - Converts simple HTML pages into visually appealing PDF reports, invoices, and posters.
- [mcanouil/awesome-quarto](https://github.com/mcanouil/awesome-quarto) - A curated list of Quarto talks, tools, examples and articles.
- [MdBook](https://rust-lang.github.io/mdBook/) - A command-line utility that generates books or documentation websites from Markdown files, focusing on simplicity with table of contents, navigation, and search functionality.
- [MkDocs Material](https://squidfunk.github.io/mkdocs-material/) - A theme for MkDocs with a modern and responsive design, customizable navigation, search, and a clean user interface.
- [MkDocs](https://www.mkdocs.org/) - A Python-based static site generator for creating documentation websites using Markdown and YAML configuration, offering themes and plugins.
- [Nextra](https://nextra.site/) - A minimalist and customizable Next.js theme for building documentation websites with Markdown and code syntax highlighting.
- [Quarto](https://quarto.org/) - A scientific computing environment integrating Markdown, code, and output into a single format, enabling reproducible reports, books, or websites with multiple language support and rich media content.
- [secretGeek/clowncar](https://github.com/secretGeek/clowncar) - A lightweight static site generator converting Markdown files into HTML pages without complex configurations or dependencies.

## EMBL-EBI

- [Tools & Data Resources](https://www.ebi.ac.uk/services) - The European Bioinformatics Institute (EMBL-EBI) maintains the world’s most comprehensive range of freely available and up-to-date molecular data resources.

## Genome alignment and comparison

- [evotools/nf-LO](https://github.com/evotools/nf-LO) - A Nextflow workflow to generate liftOver files for any pair of genomes.
- [gamcil/clinker](https://github.com/gamcil/clinker) - Gene cluster comparison figure generator.
- [lastz/lastz](https://github.com/lastz/lastz) - A program for aligning DNA sequences, a pairwise aligner.
- [mauve](https://darlinglab.org/mauve/mauve.html) - A system for efficiently constructing multiple genome alignments in the presence of large-scale evolutionary events such as rearrangement and inversion.
- [metagenlab/mummer2circos](https://github.com/metagenlab/mummer2circos) - Circular bacterial genome plots based on BLAST or NUCMER/PROMER alignments.
- [moshi4/ANIclustermap](https://github.com/moshi4/ANIclustermap) - A tool for clustering genomes based on ANI values.
- [mummer4/mummer](https://github.com/mummer4/mummer) - A versatile and fast alignment tool for DNA and protein sequences that can align mammalian genomes in a few hours.
- [ParBLiSS/FastANI](https://github.com/ParBLiSS/FastANI) - Fast alignment-free computation of whole-genome average nucleotide identity (ANI).
- [paulstothard/cgview_comparison_tool](https://github.com/paulstothard/cgview_comparison_tool) - A package for visually comparing bacterial, plasmid, chloroplast, and mitochondrial sequences.
- [schneebergerlab/plotsr](https://github.com/schneebergerlab/plotsr) - Plot synteny and structural rearrangements between genomes.
- [schneebergerlab/syri](https://github.com/schneebergerlab/syri) - Predict and visualize genomic differences between related genomes using whole-genome assemblies.

## Genome annotation and sequence characterization

- [AdmiralenOla/Scoary](https://github.com/AdmiralenOla/Scoary) - Pan-genome wide association studies.
- [fmalmeida/bacannot](https://github.com/fmalmeida/bacannot) - Generic but comprehensive pipeline for prokaryotic genome annotation and interrogation with interactive reports.
- [fwhelan/coinfinder](https://github.com/fwhelan/coinfinder) - A tool for the identification of coincident (associating and dissociating) genes in pangenomes.
- [Gaius-Augustus/BRAKER](https://github.com/Gaius-Augustus/BRAKER) - A pipeline for fully automated prediction of protein coding gene structures with GeneMark-ES/ET and AUGUSTUS in novel eukaryotic genomes.
- [gbouras13/pharokka](https://github.com/gbouras13/pharokka) - Fast phage annotation.
- [gtonkinhill/panaroo](https://github.com/gtonkinhill/panaroo) - Producing polished prokaryotic pangenomes.
- [jime-sg/deleat](https://github.com/jime-sg/deleat) - Gene essentiality prediction and deletion design for bacterial genome reduction.
- [jotech/gapseq](https://github.com/jotech/gapseq) - Informed prediction and analysis of bacterial metabolic pathways and genome-scale networks.
- [marcelauliano/MitoHiFi](https://github.com/marcelauliano/MitoHiFi) - Assembles mitogenomes from Pacbio HiFi reads.
- [ncbi/amr](https://github.com/ncbi/amr) - Identifies AMR genes, resistance-associated point mutations, and select other classes of genes using protein annotations and/or assembled nucleotide sequence.
- [ncbi/pgap](https://github.com/ncbi/pgap) - Annotate bacterial and archaeal genomes (chromosomes and plasmids).
- [nextgenusfs/funannotate](https://github.com/nextgenusfs/funannotate) - Eukaryotic genome annotation pipeline.
- [oschwengers/bakta](https://github.com/oschwengers/bakta) - A tool for the rapid and standardized annotation of bacterial genomes and plasmids from isolates and MAGs.
- [replikation/What_the_Phage](https://github.com/replikation/What_the_Phage) - A scalable and easy-to-use workflow for phage identification and analysis.
- [sanger-pathogens/Roary](https://github.com/sanger-pathogens/Roary) - Rapid large-scale prokaryote pan genome analysis.
- [soedinglab/metaeuk](https://github.com/soedinglab/metaeuk) - A modular toolkit designed for large-scale gene discovery and annotation in eukaryotic metagenomic contigs.
- [tseemann/mlst](https://github.com/tseemann/mlst) - Scan contig files against traditional PubMLST typing schemes.
- [tseemann/prokka](https://github.com/tseemann/prokka) - Rapid prokaryotic genome annotation.
- [WrightonLabCSU/DRAM](https://github.com/WrightonLabCSU/DRAM) - Distilled and Refined Annotation of Metabolism: A tool for the annotation and curation of function for microbial and viral genomes.

## Genome assembly

- [ablab/quast](https://github.com/ablab/quast) - Evaluates genome/metagenome assemblies by computing various metrics.
- [ablab/spades](https://github.com/ablab/spades) - SPAdes genome assembler.
- [adigenova/wengan](https://github.com/adigenova/wengan) - An accurate and ultra-fast hybrid genome assembler.
- [alekseyzimin/masurca](https://github.com/alekseyzimin/masurca) - The MaSuRCA (Maryland Super Read Cabog Assembler) genome assembly and analysis toolkit.
- [BUSCO](https://busco.ezlab.org) - Evaluate the completeness of a genome assembly and annotation by assessing the presence of a set of single-copy, conserved orthologous genes.
- [Ecogenomics/CheckM](https://github.com/Ecogenomics/CheckM) - Assess the quality of genomes recovered from isolates, single cells, or metagenomes.
- [fenderglass/Flye](https://github.com/fenderglass/Flye) - Fast and accurate de novo assembler for single molecule sequencing reads.
- [gbouras13/hybracter](https://github.com/gbouras13/hybracter) - An automated long-read first bacterial genome assembly tool implemented in Snakemake.
- [gbouras13/plassembler](https://github.com/gbouras13/plassembler) - Program to quickly and accurately assemble plasmids in hybrid and long-only sequenced bacterial isolates.
- [Kinggerm/GetOrganelle](https://github.com/Kinggerm/GetOrganelle) - A fast and versatile toolkit for accurate assembly of organelle genomes.
- [lbcb-sci/racon](https://github.com/lbcb-sci/racon) - Ultrafast consensus module for raw de novo genome assembly of long uncorrected reads.
- [lbcb-sci/raven](https://github.com/lbcb-sci/raven) - A de novo genome assembler for long uncorrected reads.
- [lh3/miniasm](https://github.com/lh3/miniasm) - Ultrafast de novo assembly for long noisy reads.
- [malonge/RagTag](https://github.com/malonge/RagTag) - Tools for fast and flexible genome assembly scaffolding and improvement.
- [marbl/canu](https://github.com/marbl/canu) - A single molecule sequence assembler for genomes large and small.
- [marbl/verkko](https://github.com/marbl/verkko) - A hybrid genome assembly pipeline developed for telomere-to-telomere assembly of PacBio HiFi and Oxford Nanopore reads.
- [nanoporetech/medaka](https://github.com/nanoporetech/medaka) - A tool to create consensus sequences and variant calls from nanopore sequencing data.
- [Nextomics/NextDenovo](https://github.com/Nextomics/NextDenovo) - Fast and accurate de novo assembler for long reads.
- [Nextomics/NextPolish](https://github.com/Nextomics/NextPolish) - Fast and accurate polishing of genomes generated by long reads.
- [rrwick/Bandage](https://github.com/rrwick/Bandage) - A tool that allows users to interact with the assembly graphs made by de novo assemblers such as Velvet, SPAdes, and MEGAHIT.
- [rrwick/Minipolish](https://github.com/rrwick/Minipolish) - Use Racon to polish a miniasm assembly, while keeping the assembly in graph form.
- [rrwick/Polypolish](https://github.com/rrwick/Polypolish) - A short-read polishing tool for long-read assemblies.
- [rrwick/Trycycler](https://github.com/rrwick/Trycycler) - A tool for generating consensus long-read assemblies for bacterial genomes.
- [rrwick/Unicycler](https://github.com/rrwick/Unicycler) - A hybrid assembly pipeline for bacterial genomes.
- [tseemann/shovill](https://github.com/tseemann/shovill) - Assemble bacterial isolate genomes from Illumina paired-end reads.
- [vgl-hub/gfastats](https://github.com/vgl-hub/gfastats) - Generate FASTA file summary statistics and manipulate FASTA files.
- [whatshap/whatshap](https://github.com/whatshap/whatshap) - Read-based phasing of genomic variants.
- [xiaochuanle/NECAT](https://github.com/xiaochuanle/NECAT) - A de novo assembly tool for Nanopore long noisy reads.

## Genome visualization

- [bernatgel/karyoploteR](https://github.com/bernatgel/karyoploteR) - An R package to plot arbitrary data along the genome.
- [cmdcolin/awesome-genome-visualization](https://github.com/cmdcolin/awesome-genome-visualization) - Interesting genome browser or genome-browser-like implementations.
- [deeptools/pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks) - Reproducible plots for multivariate genomic data sets.
- [igvteam/igv-reports](https://github.com/igvteam/igv-reports) - Generate self-contained HTML reports that consist of a table of genomic sites or regions and associated IGV views for each site.
- [moshi4/pyGenomeViz](https://github.com/moshi4/pyGenomeViz) - A genome visualization Python package for comparative genomics.
- [paulstothard/cgview](https://github.com/paulstothard/cgview) - Generate high-quality, zoomable maps of circular genomes.
- [Proksee](https://proksee.ca) - In-depth characterization and visualization of bacterial genomes.

## GTF and GFF

- [agshumate/Liftoff](https://github.com/agshumate/Liftoff) - A tool that accurately maps annotations in GFF or GTF between assemblies of the same, or closely-related species.
- [gpertea/gffread](https://github.com/gpertea/gffread) - GFF/GTF utility providing format conversions, region filtering, FASTA sequence extraction and more.
- [NBISweden/AGAT](https://github.com/NBISweden/AGAT) - A suite of tools to handle gene annotations in any GTF/GFF format.

## GWAS

- [brentp/vcfassoc](https://github.com/brentp/vcfassoc) - Perform genotype-phenotype-association tests on a VCF with logistic regression.
- [chrchang/plink-ng](https://github.com/chrchang/plink-ng) - A comprehensive update to the PLINK association analysis toolset.
- [jianyangqt/gcta](https://github.com/jianyangqt/gcta) - A software package for performing genome-wide association studies and many related analyses.
- [MareesAT/GWA_tutorial](https://github.com/MareesAT/GWA_tutorial) - A comprehensive tutorial about GWAS and PRS.
- [TASSEL](https://www.maizegenetics.net/tassel) - A software package for assessing diversity, linkage disequilibrium, relatedness, and genotype / phenotype associations.
- [xiaolei-lab/rMVP](https://github.com/xiaolei-lab/rMVP) - A Memory-efficient, visualization-enhanced, and parallel-accelerated tool for GWAS.

## Images

- [BioRender](https://biorender.com) - Create professional science figures in minutes.
- [faressoft/terminalizer](https://github.com/faressoft/terminalizer) - Record your terminal and generate animated gif images or share a web player.
- [flameshot-org/flameshot](https://github.com/flameshot-org/flameshot) - Powerful yet simple to use screenshot software.
- [GIMP](https://www.gimp.org) - A free and open-source raster graphics editor.
- [ImageJ](https://imagej.net/ij/index.html) - A Java-based image processing program developed at the National Institutes of Health.
- [ImageMagick](https://imagemagick.org) - A free and open-source software suite for displaying, converting, and editing raster image and vector image files.
- [nbedos/termtosvg](https://github.com/nbedos/termtosvg) - Record terminal sessions as SVG animations.
- [Photopea](https://www.photopea.com) - A free online tool for editing raster and vector graphics with support for PSD, AI, and Sketch files.
- [sindresorhus/pageres-cli](https://github.com/sindresorhus/pageres-cli) - Capture website screenshots.

## Machine learning

- [DeepLabCut/DeepLabCut](https://github.com/DeepLabCut/DeepLabCut) - Markerless pose estimation of user-defined features with deep learning for all animals.
- [josephmisiti/awesome-machine-learning](https://github.com/josephmisiti/awesome-machine-learning) - A curated list of awesome machine learning frameworks, libraries and software.
- [karpathy/convnetjs](https://github.com/karpathy/convnetjs) - A JavaScript implementation of neural networks.
- [keras-team/keras](https://github.com/keras-team/keras) - A deep learning API written in Python, running on top of the machine learning platform TensorFlow.
- [marrlab/InstantDL](https://github.com/marrlab/InstantDL) - An easy and convenient deep learning pipeline for image segmentation and classification.
- [neural networks course](https://karpathy.ai/zero-to-hero.html) - A course by Andrej Karpathy on building neural networks, from scratch, in code.
- [pytorch/pytorch](https://github.com/pytorch/pytorch) - Tensors and Dynamic neural networks in Python with strong GPU acceleration.
- [scikit-learn/scikit-learn](https://github.com/scikit-learn/scikit-learn) - Machine learning in Python.
- [tensorflow/tensorflow](https://github.com/tensorflow/tensorflow) - An end-to-end open source platform for machine learning. It has a comprehensive, flexible ecosystem of tools, libraries, and community resources that lets researchers push the state-of-the-art in ML and developers easily build and deploy ML-powered applications.

## Metagenomics

- [biobakery/biobakery_workflows](https://github.com/biobakery/biobakery_workflows) - A collection of workflows and tasks for executing common microbial community analyses using standardized, validated tools and parameters.
- [biobakery/humann](https://github.com/biobakery/humann) - A pipeline for efficiently and accurately profiling the presence/absence and abundance of microbial pathways in a community from metagenomic or metatranscriptomic sequencing data.
- [biobakery/Maaslin2](https://github.com/biobakery/Maaslin2) - A comprehensive R package for efficiently determining multivariable association between clinical metadata and microbial meta-omics features.
- [biobakery/MetaPhlAn](https://github.com/biobakery/MetaPhlAn) - A computational tool for profiling the composition of microbial communities (Bacteria, Archaea and Eukaryotes) from metagenomic shotgun sequencing data.
- [Ecogenomics/GTDBTk](https://github.com/Ecogenomics/GTDBTk) - A toolkit for assigning objective taxonomic classifications to bacterial and archaeal genomes.
- [fbreitwieser/pavian](https://github.com/fbreitwieser/pavian) - Interactive analysis of metagenomics data.
- [metagenome-atlas/atlas](https://github.com/metagenome-atlas/atlas) - Metagenome-Atlas is a easy-to-use metagenomic pipeline based on Snakemake.
- [MrOlm/drep](https://github.com/MrOlm/drep) - Rapid comparison and dereplication of genomes.
- [nf-core/mag](https://github.com/nf-core/mag) - Assembly and binning of metagenomes.

## Methyl-seq

- [EpiDiverse](https://github.com/EpiDiverse) - A collection of Nextflow pipelines for epigenome analysis.
- [kdkorthauer/dmrseq](https://github.com/kdkorthauer/dmrseq) - R package for inference of differentially methylated regions (DMRs) from bisulfite sequencing.
- [nf-core/methylseq](https://github.com/nf-core/methylseq) - Methylation (Bisulfite-Sequencing) analysis pipeline using Bismark or bwa-meth + MethylDackel.

## Multiomics

- [bioFAM/MOFA2](https://github.com/bioFAM/MOFA2) - A factor analysis model that provides a general framework for the integration of multiomic data sets in an unsupervised fashion.
- [cantinilab/momix-notebook](https://github.com/cantinilab/momix-notebook) - Evaluation of multiomics joint dimensionality reduction approaches.
- [mixOmics](https://www.bioconductor.org/packages/release/bioc/html/mixOmics.html) - An R package that offers a wide range of multivariate methods for the exploration and integration of biological datasets with a particular focus on variable selection.

## NCBI

- [All Resources](https://www.ncbi.nlm.nih.gov/guide/all/) - The National Center for Biotechnology Information (NCBI) advances science and health by providing access to biomedical and genomic information.
- [NCBI-Hackathons/EDirectCookbook](https://github.com/NCBI-Hackathons/EDirectCookbook) - Examples illustrating the use of NCBI's Entrez Direct (EDirect), which provides access to the NCBI's suite of interconnected databases.
- [shenwei356/taxonkit](https://github.com/shenwei356/taxonkit) - A practical and efficient NCBI Taxonomy toolkit.

## Phylogenetics and phylogenomics

- [achtman-lab/GrapeTree](https://github.com/achtman-lab/GrapeTree) - A fully interactive, tree visualization program, which supports facile manipulations of both tree layout and metadata.
- [AstrobioMike/GToTree](https://github.com/AstrobioMike/GToTree) - A user-friendly workflow for phylogenomics.
- [biobakery/phylophlan](https://github.com/biobakery/phylophlan) - Precise phylogenetic analysis of microbial isolates and genomes from metagenomes.
- [Cibiv/IQ-TREE](https://github.com/Cibiv/IQ-TREE) - Efficient software for phylogenomic inference.
- [davidemms/OrthoFinder](https://github.com/davidemms/OrthoFinder) - Finds orthogroups and orthologs, infers rooted gene trees for all orthogroups and identifies all of the gene duplication events in those gene trees.
- [FastTree](http://www.microbesonline.org/fasttree/) - Efficient in memory and time inference of phylogenetic trees from up to a million aligned nucleotide or protein sequences.
- [IcyTree](https://icytree.org) - A browser-based phylogenetic tree viewer.
- [iTOL](https://itol.embl.de) - iTOL is an online tool for the display, annotation, and management of phylogenetic trees.
- [MEGA](https://www.megasoftware.net) - Conduct statistical analysis of molecular evolution and for constructing phylogenetic trees.
- [stephaneguindon/phyml](https://github.com/stephaneguindon/phyml) - Package that uses modern statistical approaches to analyse alignments of nucleotide or amino acid sequences in a phylogenetic framework.
- [YuLab-SMU/ggtree](https://github.com/YuLab-SMU/ggtree) - Visualization and annotation of phylogenetic trees.

## Population genetics and conservation

- [ADMIXTURE](https://dalexander.github.io/admixture/) - A software tool for maximum likelihood estimation of individual ancestries from multilocus SNP genotype datasets.
- [ANGSD/angsd](https://github.com/ANGSD/angsd) - Perform population genetic analyses using raw sequencing data or by using genotype likelihoods.
- [Cervus](http://www.fieldgenetics.com/pages/aboutCervus_Overview.jsp) - A tool for parentage analysis that supports microsatellite and SNP data.
- [COLONY](https://www.zsl.org/about-zsl/resources/software/colony) - Can be used in estimating full- and half-sib relationships, assigning parentage, inferring mating system (polygamous/monogamous) and reproductive skew in both diploid and haplo-diploid species.
- [fastStructure](https://rajanil.github.io/fastStructure/) - Infer population structure from large SNP genotype data.
- [HAPMIX](https://www.hsph.harvard.edu/alkes-price/software/) - Infer chromosomal segments of distinct continental ancestry in admixed populations, using dense genetic data.
- [lima1/franzpedigree](https://github.com/lima1/franzpedigree) - A fast and flexible parentage inference program for natural populations.
- [millanek/Dsuite](https://github.com/millanek/Dsuite) - Fast calculation of Patterson's D (ABBA-BABA) and the f4-ratio statistics across many populations/species using VCF files and genotype uncertainty.
- [NGSadmix](http://www.popgen.dk/software/index.php/NgsAdmix) - A tool for estimating individual admixture proportions from NGS data that makes use of genotype likelihoods and works well for medium and low coverage NGS data.
- [STRUCTURE](https://web.stanford.edu/group/pritchardlab/structure.html) - Use multi-locus genotype data to investigate population structure.
- [thibautjombart/adegenet](https://github.com/thibautjombart/adegenet) - An R package package for the exploratory analysis of genetic data.

## Programming

- [Automate the Boring Stuff with Python](https://automatetheboringstuff.com) - Practical programming for total beginners.
- [gto76/python-cheatsheet](https://github.com/gto76/python-cheatsheet) - A comprehensive Python cheat sheet.
- [IPGP/scientific_python_cheat_sheet](https://github.com/IPGP/scientific_python_cheat_sheet) - A quick reference for scientific computing with Python.
- [R for Data Science](https://r4ds.had.co.nz) - Learn how to get your data into R, get it into the most useful structure, transform it, visualize it and model it.
- [ralish/bash-script-template](https://github.com/ralish/bash-script-template) - A best practices template for bash scripts.
- [The Modern JavaScript Tutorial](https://javascript.info) - From the basics to advanced topics with simple, but detailed explanations.
- [the-art-of-command-line](https://github.com/jlevy/the-art-of-command-line) - Master the command line, in one page.
- [awesome-vscode](https://github.com/viatsko/awesome-vscode) - A curated list of delightful VS Code packages and resources.
- [ziishaned/learn-regex](https://github.com/ziishaned/learn-regex) - Learn regex the easy way.

## Raw sequence data processing and QC

- [BBMap](https://sourceforge.net/projects/bbmap/) - Includes BBMap, a short read aligner, as well as various other bioinformatic tools written in Java.
- [gear-genomics/tracy](https://github.com/gear-genomics/tracy) - Basecalling, alignment, assembly and deconvolution of Sanger Chromatogram trace files.
- [huishenlab/biscuit](https://github.com/huishenlab/biscuit) - Perform alignment, DNA methylation and mutation calling, and allele specific methylation from bisulfite sequencing data.
- [kishwarshafin/pepper](https://github.com/kishwarshafin/pepper) - A genome inference module based on recurrent neural networks that enables long-read variant calling and nanopore assembly polishing in the PEPPER-Margin-DeepVariant pipeline.
- [lh3/seqtk](https://github.com/lh3/seqtk) - Toolkit for processing sequences in FASTA/Q formats.
- [OpenGene/fastp](https://github.com/OpenGene/fastp) - An ultra-fast all-in-one FASTQ preprocessor (QC/adapters/trimming/filtering/splitting/merging).
- [rrwick/Filtlong](https://github.com/rrwick/Filtlong) - Quality filtering tool for long reads.
- [s-andrews/FastQC](https://github.com/s-andrews/FastQC) - A quality control application for high throughput sequence data.
- [sheinasim/HiFiAdapterFilt](https://github.com/sheinasim/HiFiAdapterFilt) - Remove adapter-contaminated PacBio HiFi reads.
- [wdecoster/chopper](https://github.com/wdecoster/chopper) - Filter and trim long reads.
- [wdecoster/NanoPlot](https://github.com/wdecoster/NanoPlot) - Plotting scripts for long read sequencing data.

## RNA-Seq

- [kevinblighe/EnhancedVolcano](https://github.com/kevinblighe/EnhancedVolcano) - Publication-ready volcano plots with enhanced colouring and labeling.
- [nf-core/nanoseq](https://github.com/nf-core/nanoseq) - Nanopore demultiplexing, QC and alignment pipeline.
- [nf-core/rnaseq](https://github.com/nf-core/rnaseq) - RNA sequencing analysis pipeline using STAR, RSEM, HISAT2 or Salmon with gene/isoform counts and extensive quality control.
- [pachterlab/kallisto](https://github.com/pachterlab/kallisto) - Near-optimal RNA-Seq quantification.
- [STAR-Fusion/STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion) - Uses the STAR aligner to identify candidate fusion transcripts supported by Illumina reads.
- [stemangiola/bioc_2020_tidytranscriptomics](https://github.com/stemangiola/bioc_2020_tidytranscriptomics) - A tidy transcriptomics introduction to RNA sequencing analyses.
- [suhrig/arriba](https://github.com/suhrig/arriba) - Fast and accurate gene fusion detection from RNA-Seq data.

## Sequence read alignment

- [lh3/bwa](https://github.com/lh3/bwa) - Burrows-Wheeler aligner for short-read alignment.
- [lh3/minimap2](https://github.com/lh3/minimap2) - A versatile pairwise aligner for genomic and spliced nucleotide sequences.
- [philres/ngmlr](https://github.com/philres/ngmlr) - A long-read mapper designed to align PacBio or Oxford Nanopore (standard and ultra-long) to a reference genome with a focus on reads that span structural variations.

## Sequence searching and clustering

- [bbuchfink/diamond](https://github.com/bbuchfink/diamond) - Accelerated BLAST compatible local sequence aligner.
- [BLAST+](https://blast.ncbi.nlm.nih.gov/doc/blast-help/downloadblastdata.html) - A suite of command-line tools to run BLAST searches.
- [soedinglab/MMseqs2](https://github.com/soedinglab/MMseqs2) - Search and cluster huge protein and nucleotide sequence sets.

## Sequence utilities

- [EMBOSS](https://emboss.sourceforge.net) - A free open source software analysis package developed for the needs of the molecular biology and bioinformatics user community.
- [ialbert/bio](https://github.com/ialbert/bio) - A collection of command-line utilities for working with sequence records.
- [kblin/ncbi-acc-download](https://github.com/kblin/ncbi-acc-download) - Download files from NCBI Entrez by accession.
- [kingfisher-download](https://github.com/wwood/kingfisher-download) - Easier download/extract of FASTA/Q read data and metadata from the ENA, NCBI, AWS or GCP.
- [lindenb/jvarkit](https://github.com/lindenb/jvarkit) - Java utilities for bioinformatics.
- [nf-core/fetchngs](https://github.com/nf-core/fetchngs) - Pipeline to fetch metadata and raw FASTQ files from public and private databases.
- [pachterlab/ffq](https://github.com/pachterlab/ffq) - A tool to find sequencing data and metadata from public databases.
- [paulstothard/sequence_manipulation_suite](https://github.com/paulstothard/sequence_manipulation_suite) - A collection of simple JavaScript programs for generating, formatting, and analyzing short DNA and protein sequences.
- [shenwei356/seqkit](https://github.com/shenwei356/seqkit) - A cross-platform and ultrafast toolkit for FASTA/Q file manipulation.
- [telatin/seqfu2](https://github.com/telatin/seqfu2) - A general-purpose program to manipulate and parse information from FASTA/FASTQ files, supporting gzipped input files.
- [tseemann/any2fasta](https://github.com/tseemann/any2fasta) - Convert various sequence formats to FASTA.

## Software development tools

- [cookiecutter/cookiecutter](https://github.com/cookiecutter/cookiecutter) - A cross-platform command-line utility that creates projects from cookiecutters (project templates), e.g. Python package projects, C projects.
- [heyman/heynote](https://github.com/heyman/heynote/) - A dedicated scratchpad for developers.
- [JupyterLab](https://jupyter.org) - A web-based interactive development environment for notebooks, code, and data.
- [nteract/papermill](https://github.com/nteract/papermill) - Parameterize, execute, and analyze Jupyter Notebooks.
- [RStudio Desktop](https://posit.co/download/rstudio-desktop/) - An integrated development environment for R, a programming language for statistical computing and graphics.
- [Visual Studio Code](https://code.visualstudio.com) - A source-code editor for Windows, Linux and macOS. Features include support for debugging, syntax highlighting, intelligent code completion, snippets, code refactoring, and embedded Git.

## Statistics

- [Design of Experiments](https://online.stat.psu.edu/stat503/) - A course on the design of experiments.
- [easystats/easystats](https://github.com/easystats/easystats) - A collection of R packages designed to provide a unifying and consistent framework to tame, discipline, and harness the scary R statistics and their pesky models.
- [easystats/report](https://github.com/easystats/report) - Automatically produces reports of R models and data frames according to best practices guidelines (e.g., APA’s style), ensuring standardization and quality in results reporting.
- [IndrajeetPatil/ggstatsplot](https://github.com/IndrajeetPatil/ggstatsplot) - Enhancing ggplot2 plots with statistical analysis.
- [jasp-stats/jasp-desktop](https://github.com/jasp-stats/jasp-desktop) - A complete statistical package for both Bayesian and frequentist statistical methods, that is easy to use and familiar to users of SPSS.
- [kassambara/factoextra](https://github.com/kassambara/factoextra) - Extract and visualize the results of multivariate data analyses.
- [paulvanderlaken/ppsr](https://github.com/paulvanderlaken/ppsr) - R implementation of Predictive Power Score.
- [statsmodels/statsmodels](https://github.com/statsmodels/statsmodels) - Statistical modeling and econometrics in Python.
- [topGO](https://bioconductor.org/packages/release/bioc/html/topGO.html) - Semi-automated enrichment analysis for Gene Ontology (GO) terms.

## Tabular data

- [apache/arrow](https://github.com/apache/arrow) - A multi-language toolbox for accelerated data interchange and in-memory processing.
- [arq5x/bedtools2](https://github.com/arq5x/bedtools2) - Intersect, merge, count, complement, and shuffle genomic intervals from multiple files in widely-used genomic file formats such as BAM, BED, GFF/GTF, VCF.
- [bedops/bedops](https://github.com/bedops/bedops) - High-performance genomic feature operations.
- [BurntSushi/xsv](https://github.com/BurntSushi/xsv) - A fast CSV command line toolkit written in Rust.
- [ClickHouse/ClickHouse](https://github.com/ClickHouse/ClickHouse) - A column-oriented database management system that allows generating analytical data reports in real-time.
- [duckdb/duckdb](https://github.com/duckdb/duckdb) - A high-performance analytical database system that provides a rich SQL dialect.
- [harelba/q](https://github.com/harelba/q) - Run SQL directly on delimited files and multi-file sqlite databases.
- [johnkerl/miller](https://github.com/johnkerl/miller) - Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON.
- [jqnatividad/qsv](https://github.com/jqnatividad/qsv) - A fork of the popular xsv command-line toolkit for working with CSV files that adds numerous useful features.
- [Kanaries/pygwalker](https://github.com/Kanaries/pygwalker) - Turn a pandas dataframe into a Tableau-style interface for visual analysis.
- [markfairbanks/tidytable](https://github.com/markfairbanks/tidytable) - tidytable is a data frame manipulation library for users who need data.table speed but prefer tidyverse-like syntax.
- [OpenRefine](https://openrefine.org) - OpenRefine (previously Google Refine) is a powerful tool for working with messy data: cleaning it; transforming it from one format into another; and extending it with web services and external data.
- [pandas-dev/pandas](https://github.com/pandas-dev/pandas) - Flexible and powerful data analysis and manipulation library for Python.
- [pstaender/csv2md](https://github.com/pstaender/csv2md) - Converts CSV data to Markdown tables.
- [ropensci/skimr](https://github.com/ropensci/skimr) - A frictionless, pipeable approach to dealing with summary statistics.
- [saulpw/visidata](https://github.com/saulpw/visidata) - A terminal spreadsheet multitool for discovering and arranging data.
- [shenwei356/csvtk](https://github.com/shenwei356/csvtk) - A cross-platform, efficient and practical CSV/TSV toolkit in Golang.
- [sqlitebrowser/sqlitebrowser](https://github.com/sqlitebrowser/sqlitebrowser) - A high quality, visual, open source tool to create, design, and edit database files compatible with SQLite.
- [wireservice/csvkit](https://github.com/wireservice/csvkit) - A suite of utilities for converting to and working with CSV, the king of tabular file formats.

## Utilities

- [adrianlopezroche/fdupes](https://github.com/adrianlopezroche/fdupes) - A program for identifying or deleting duplicate files residing within specified directories.
- [alacritty/alacritty](https://github.com/alacritty/alacritty) - A cross-platform, GPU-accelerated terminal emulator.
- [axel-download-accelerator/axel](https://github.com/axel-download-accelerator/axel) - Tries to accelerate the download process by using multiple connections per file, and can also balance the load between different servers. Supports HTTP, HTTPS, FTP and FTPS protocols.
- [Clipy/Clipy](https://github.com/Clipy/Clipy) - Clipboard extension app for macOS.
- [divriots/jampack](https://github.com/divriots/jampack) - Optimizes static websites to improve performance and user experience.
- [gildas-lormeau/SingleFile](https://github.com/gildas-lormeau/SingleFile) - Web extension for saving a faithful copy of a complete web page in a single HTML file.
- [ibraheemdev/modern-unix](https://github.com/ibraheemdev/modern-unix) - A collection of modern alternatives to common Unix commands.
- [joh/when-changed](https://github.com/joh/when-changed) - Execute a command when a file is changed.
- [jonschlinkert/markdown-toc](https://github.com/jonschlinkert/markdown-toc) - API and CLI for adding a table of contents to a Markdown file.
- [phiresky/ripgrep-all](https://github.com/phiresky/ripgrep-all) - Wraps ripgrep and enables it to search more file types.
- [rclone/rclone](https://github.com/rclone/rclone) - A command-line program to sync files and directories to and from different cloud storage providers.
- [restic/restic](https://github.com/restic/restic) - Fast, secure, efficient backup program.
- [schollz/croc](https://github.com/schollz/croc) - Easily and securely send things from one computer to another.
- [stevenvachon/broken-link-checker](https://github.com/stevenvachon/broken-link-checker) - Find broken links within HTML files.
- [tcort/markdown-link-check](https://github.com/tcort/markdown-link-check) - Check hyperlinks in Markdown text.
- [Y2Z/monolith](https://github.com/Y2Z/monolith) - Command-line tool to save web pages as a single HTML file.

## Variant identification and analysis

- [ACEnglish/truvari](https://github.com/ACEnglish/truvari) - A toolkit for benchmarking, merging, and annotating structural variants.
- [barricklab/breseq](https://github.com/barricklab/breseq) - A computational pipeline for finding mutations relative to a reference sequence in short-read DNA resequencing data. It is intended for haploid microbial genomes.
- [CRG-CNAG/CalliNGS-NF](https://github.com/CRG-CNAG/CalliNGS-NF) - GATK RNA-Seq variant calling in Nextflow.
- [HKU-BAL/Clair3](https://github.com/HKU-BAL/Clair3) - A germline small variant caller for long-reads.
- [fritzsedlazeck/Sniffles](https://github.com/fritzsedlazeck/Sniffles) - A fast structural variant caller for long-read sequence data.
- [fritzsedlazeck/SURVIVOR](https://github.com/fritzsedlazeck/SURVIVOR) - Toolset for SV simulation, comparison and filtering.
- [marbl/gingr](https://github.com/marbl/gingr) - A flexible platform for visualizing and compressing alignments and phylogenetic trees.
- [marbl/parsnp](https://github.com/marbl/parsnp) - A command-line tool for efficient microbial core genome alignment and SNP detection.
- [mkirsche/Jasmine](https://github.com/mkirsche/Jasmine) - A pipeline for accurately detecting SVs and comparing variant calls across large numbers of individuals.
- [nf-core/sarek](https://github.com/nf-core/sarek) - Analysis pipeline to detect germline or somatic variants (pre-processing, variant calling and annotation) from WGS / targeted sequencing.
- [PacificBiosciences/pbsv](https://github.com/PacificBiosciences/pbsv) - PacBio structural variant calling and analysis tools.
- [PoisonAlien/maftools](https://github.com/PoisonAlien/maftools) - Summarize, analyze and visualize MAF files from TCGA or in-house studies.
- [ryanlayer/samplot](https://github.com/ryanlayer/samplot) - Plot structural variant signals from many BAMs and CRAMs.
- [tjiangHIT/cuteSV](https://github.com/tjiangHIT/cuteSV) - Long-read structural variation detection.
- [tseemann/snippy](https://github.com/tseemann/snippy) - Rapid haploid variant calling and core genome alignment.

## VCF files

- [BGI-shenzhen/VCF2Dis](https://github.com/BGI-shenzhen/VCF2Dis) - A simple and efficient tool to calculate a p-distance matrix from VCF files.
- [brentp/vcfanno](https://github.com/brentp/vcfanno) - Annotate a VCF with other VCFs/BEDs/tabixed files.
- [knausb/vcfR](https://github.com/knausb/vcfR) - A package to manipulate and visualize VCF data in R.
- [pcingola/SnpEff](https://github.com/pcingola/SnpEff) - Genomic variant annotations and functional effect prediction toolbox.
- [samtools/bcftools](https://github.com/samtools/bcftools) - A set of utilities that manipulate variant calls in the Variant Call Format (VCF) and its binary counterpart BCF. All commands work transparently with both VCFs and BCFs, both uncompressed and BGZF-compressed.
- [vcflib/vcflib](https://github.com/vcflib/vcflib) - C++ library and command-line tools for parsing and manipulating VCF files.
- [vcftools/vcftools](https://github.com/vcftools/vcftools) - A set of tools written in Perl and C++ for working with VCF files.
- [vembrane/vembrane](https://github.com/vembrane/vembrane) - Filter VCF files using Python expressions.

## Vim

- [iggredible/Learn-Vim](https://github.com/iggredible/Learn-Vim) - Vim guide for beginner and advanced users.

## Workflow development and workflows

- [bcbio/bcbio-nextgen](https://github.com/bcbio/bcbio-nextgen) - A Python toolkit and pipelines for fully automated high throughput sequence data analysis.
- [maxplanck-ie/snakepipes](https://github.com/maxplanck-ie/snakepipes) - Customizable workflows based on snakemake and python for the analysis of NGS data.
- [nextflow-io/nextflow](https://github.com/nextflow-io/nextflow) - A bioinformatics workflow manager that enables the development of portable and reproducible workflows.
- [nf-core/tools](https://github.com/nf-core/tools) - Tools for working with nf-core pipelines.
- [nfcore](https://nf-co.re) - A community effort to collect a curated set of analysis pipelines built using Nextflow.
- [Nukesor/pueue](https://github.com/Nukesor/pueue) - A command-line task queue for sequential and parallel execution of long-running tasks.
- [ploomber/ploomber](https://github.com/ploomber/ploomber) - A framework to build collaborative and modular pipelines.
- [ropensci/targets](https://github.com/ropensci/targets) - A Make-like pipeline tool for statistics and data science in R.
- [Snakemake workflow catalog](https://snakemake.github.io/snakemake-workflow-catalog) - A comprehensive catalog of standards compliant, public, Snakemake workflows.
- [snakemake/snakemake-wrappers](https://github.com/snakemake/snakemake-wrappers) - A collection of reusable wrappers for adding popular command-line tools to Snakemake workflows.
- [snakemake/snakemake](https://github.com/snakemake/snakemake) - A tool to create reproducible and scalable data analyses.
